guide/manipulating-with-obs-sequence-tool.rst

```diff
@@ -0,0 +1,16 @@
+Manipulating obs_seq files with the obs_sequence_tool
+=====================================================
+
+First and foremost, check out the
+`obs_sequence_tool.html <../assimilation_code/programs/obs_sequence_tool/obs_sequence_tool.html>`__
+document for detailed information and examples.
```

Comment:
I'm not exactly sure what added value this file has ... why not just reference the obs_sequence_tool directly from the nav bar?   Does this help somehow with the 'previous', 'next' navigation?